client/src/components/registration/RegistrationForm.js

```diff
@@ -89,6 +105,26 @@ const RegistrationForm = () => {
     <div className="grid-container">
       <h1>Register</h1>
       <form onSubmit={onSubmit}>
+        <div>
+          <label>First name
+            <input type="text"
+            name="firstName"
+            value={userPayload.firstName}
+            onChange={onInputChange}
```

Comment:
all props should be indented in. In this case, the input field properties (e.g. lines 111 to 113) should be indented in